sdcm/nemesis.py

```diff
@@ -1645,12 +1645,6 @@ def execute_disrupt_method(self, disrupt_method):
         self.metrics_srv.event_start(disrupt_method_name)
         try:
             disrupt_method()
-        except Exception as exc:  # pylint: disable=broad-except
-            error_msg = "Exception in random_disrupt_method %s: %s", disrupt_method_name, exc
-            self.log.error(error_msg, exc_info=True)
-            self.error_list.append(error_msg)
-            raise
-        else:
```

Comment:
while removing it, you are not raising it anymore... is that intentional?

Reply:
I'm not catching any exception, so they would be raised automatically